spec/features/graph/legend_spec.rb

```diff
@@ -15,8 +15,18 @@
         expect(page).to have_content "prometheus_targetpool_duration_ms"
       end
     end
+
+    it "maintains legend state after update" do
+      expect(all('.rickshaw_legend .line.disabled').count).to eq 0
+      legend_elements = all('.rickshaw_legend .action')
+      legend_elements.first.click
+      expect(all('.rickshaw_legend .line.disabled').count).to eq 1
+      open_tab 'Refresh'
```

Comment:
`open_tab` just finds the 'Refresh' on the graph and clicks it. I added in 3sec sleep statements to make sure the update had occurred but then removed them. Would you like me to add in a 100ms sleep between clicking refresh and checking for .line.disabled? Since the "prometheus" server in the feature specs is just a local sinatra server the response is almost immediate when refreshing.